pkgs/development/tools/kube-linter/default.nix

```
@@ -1,22 +1,31 @@
-{ lib, buildGoModule, fetchFromGitHub, testers, kube-linter }:
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles, testers, kube-linter }:
 
 buildGoModule rec {
   pname = "kube-linter";
-  version = "0.3.0";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
     owner = "stackrox";
     repo = pname;
     rev = version;
-    sha256 = "ZqnD9zsh+r1RL34o1nAkvO1saKe721ZJ2+DgBjmsH58=";
+    sha256 = "XAsPbl9fqYk2nhDxRg5wyPwciwSpfigoBZ4hzdWAVgw=";
   };
 
-  vendorSha256 = "sha256-tm1+2jsktNrw8S7peJz7w8k3+JwAYUgKfKWuQ8zIfvk=";
+  vendorSha256 = "sha256-0bjAIHSjw0kHrh9CzJHv1UAaBJDn6381055eOHufvCw=";
+
+  nativeBuildInputs = [ installShellFiles ];
```

```suggestion
  nativeBuildInputs = [ installShellFiles ];
```
nativeBuildInputs is `buildPhase`, it should be after `ldflags` which is `configurePhase`.